### ISSUE TITLE
The text of icon didn't display when open the "High Contrast"

### ISSUE THREAD

**Issue by yanlxu** [Microsoft/pxt#3415]
Repro steps:
• Navigate to https://makecode.microbit.org/beta
• Click "Projects" -> "Examples" ->"Plot Acceleration" 
• Switch between "Blocks" and "JavaScript"
• Click "Setting" -> "High Contrast On"

Expected result:
1. The display of icon "show data simulator" should be stable when switch between "Blocks" and "JavaScript"
2. The text of icon "show data simulator" should display when open the "High Contrast"
 
Actual result:
1. The display of icon "show data simulator" is not stable when switch between "Blocks" and "JavaScript"
2. The text of icon "show data simulator" didn't display when open the "High Contrast"

![plot acceleration](https://user-images.githubusercontent.com/11880996/32365940-f93a573c-c0b6-11e7-8f5e-7611beaa4cb5.gif)
